HabitLogger.sln

```diff
@@ -0,0 +1,31 @@
+﻿
```

Comment:
Could you please move the entire project to a folder named SinghxRaj.HabitTracker as per the code review article? 😁
![image](https://user-images.githubusercontent.com/108711476/212804744-f3c75353-bff2-4fa0-92ce-58232a2dddfe.png)
https://www.thecsharpacademy.com/article/52

Reply:
Alright, I moved all the entire project into a folder named SinghxRaj.HabitTracker.